tensorpack/train/input_source.py

```diff
@@ -5,7 +5,7 @@
 
 import tensorflow as tf
 try:
-    from tensorflow.contrib.staging import StagingArea
+    from tensorflow.python.ops.data_flow_ops import StagingArea
 except ImportError:
     pass
```

Comment:
I really enjoy this empty catch block. I know somebody who had some issues with this line.